auto_nag/scripts/not_landed.py

```diff
@@ -300,21 +325,39 @@ def get_bugs(self, date="today", bug_ids=[]):
         bugs = self.filter_bugs(bugs)
         bugs_patch = self.get_patch_data(bugs)
         res = {}
+
+        reviewers_phid = set()
         nicknames = {}
+        for bugid, data in bugs_patch.items():
+            reviewers_phid |= data["reviewers_phid"]
+            assignee = bugs[bugid]["assigned_to"]
+            if not assignee:
+                assignee = max(data["author"], key=data["author"].get)
+                nicknames[bugid] = assignee
+
+        bz_reviewers = self.get_bz_userid(reviewers_phid)
+        nicknames = self.get_nicks(nicknames)
+
         for bugid, data in bugs_patch.items():
             res[bugid] = d = bugs[bugid]
             self.extra_ni[bugid] = data["count"]
             assignee = d["assigned_to"]
             nickname = d["nickname"]
+
             if not assignee:
-                assignee = max(data["author"], key=data["author"].get)
-                nicknames[bugid] = assignee
-            else:
-                self.add_auto_ni(bugid, {"mail": assignee, "nickname": nickname})
+                assignee, nickname = nicknames[bugid]
 
-        nicknames = self.get_nicks(nicknames)
-        for bugid, (name, nick) in nicknames.items():
-            self.add_auto_ni(bugid, {"mail": name, "nickname": nick})
+            if not assignee:
+                continue
+
+            self.add_auto_ni(bugid, {"mail": assignee, "nickname": nickname})
+
+            for i in range(10):
```

Comment:
Why 10?

Reply:
Why not ?

Reply:
Let's make it a config, it's nicer than totally hardcoding it

Reply:
No, no, I removed that stuff ;)